docs/community/20230104-mixed-clusters.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Feature Group for Mixed Clusters in Cluster API
+authors:
+  - "@prometherion"
+  - "@lentzi90"
+reviewers:
+  - "@lentzi90"
+  - "@richardcase"
+  - "@fabriziopandini"
+creation-date: 2023-01-04
+last-updated: 2023-01-20
+status: proposed
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api/issues/7475#issuecomment-1302454745
+  - https://kubernetes.slack.com/archives/C8TSNPY4T/p1667458956589639
+---
+# Mixed Clusters in Cluster API Feature Group
+
+This document briefly outlines the scope, communication media, and stakeholders for a formal Feature Group dedicated to defining a Cluster API-approved solution for "mixed clusters" which combines two or more infrastructure providers.
+
+## User Story and Problem Statement
+
+As Kubernetes cluster administrator, I want to combine multiple infrastructure providers when creating a cluster (e.g. one per node pool), for efficiency and for unique abilities.
+
+We aim to extend and build on the great work done so far by providers, making it possible to use different infrastructure providers in the same cluster as well as ensuring a consistent UX when using different combinations of providers.
+
+A practical example is offered with [Kamaji](https://github.com/clastix/kamaji) that provides virtualized Control Plane resources in a pre-existing Kubernetes cluster: such control planes are then used by worker nodes provisioned on any cloud provider by using the `kubeadm` bootstrapper.
+
+Another example is the [CAPO](https://github.com/kubernetes-sigs/cluster-api-provider-openstack) for the Control Plane component and Auto Scaling node pool, along with [Metal3](https://metal3.io/) for bare metal needs.
+
+## Scope
+
+The scope of this group is to Write proposal(s) and engage with the community to reach consensus around how to support this feature.
+
+## Communication
+
+We will bi-weekly meet on [Tuesday at 10:00 AM (CET)][https://zoom.us/j/861487554]. [Convert to your timezone](https://dateful.com/time-zone-converter?t=10:00&tz=CET%20%28Central%20European%20Time%29). Meeting notes will be documented in this [Google Doc](https://docs.google.com/document/d/12TGubbGJa3w4ux8kJnR7sb4l39iHxcOAtbJqLwSlnkc/edit?usp=sharing).
```

Comment:
Are these invites being sent out to the mailing list as well?

Reply:
Would be great having more feedback from the CAPI community, still a newbie in how communications are managed.

@fabriziopandini @richardcase may I ask for your help on this?

Reply:
Until now we are promoting those features groups in the CAPI office hours only, but it is really up to the team to promote theirself.